src/plugins/cookie.svg.source

```diff
@@ -0,0 +1 @@
+https://publicdomainvectors.org/en/free-clipart/Simple-illustration-of-a-cookie/63902.html
```

Comment:
```suggestion
https://openclipart.org/detail/249534/cookie
```

Reply:
Ah, I did not see that there is a source linked. I could change this, however, the opelclipart site is [not available](https://downforeveryoneorjustme.com/openclipart.org?proto=http). Therefore I would keep the URL which I actually used to download the image.

Reply:
Yeah, though publicdomainvectors is not the original source. I'm not sure how far/deep we ought to go in terms of identifying the original author. @sgiehl what's your opinion here?

Reply:
I haven't been involved in this part too much. Guess that would be something @Findus23 would need to answer.

Reply:
Sorry, I only now saw the notification. 

The openclipart website seems to work now, so that seems fine by me.

Thanks for improving the icon!